src/koala/__init__.py

```diff
@@ -3593,7 +3592,7 @@ def __init__(
             plt.title(
                 self.object
```

Comment:
Make this whole thing a string with format

Reply:
This have been reserved for later.